src/components/quotes.js

```diff
@@ -31,12 +31,16 @@ function Quotes() {
   if (isLoading) return <div>Loading...</div>
 console.log(data.map)
   return (
```

Comment:
**[OPTIONAL]**

- [ ] Kindly remove this console to make your code clean and to follow JavaScript best practices :+1: because when I click on the quotes link on the navigation bar then I see an object in the console window :thinking: it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website :fearful: